hugegraph-llm/src/hugegraph_llm/models/llms/openai.py

```diff
@@ -141,7 +140,7 @@ def num_tokens_from_string(self, string: str) -> int:
     def max_allowed_token_length(self) -> int:
         """Get max-allowed token length"""
         # TODO: list all models and their max tokens from api
-        return 2049
+        return 8192
```

Comment:
why increase to 8192？

Reply:
> why increase to 8192？

`2049` is outdated (in 2023y)

BTW, this method is not used now (may remove it later)